jax/linear_util.py

```diff
@@ -209,6 +210,8 @@ def memoized_fun(fun, *args):
       ans = call(fun, *args)
       cache[key] = (ans, fun.stores)
     return ans
+
+  memoized_fun.cache_clear = lambda: fun_caches.clear()
```

Comment:
Can you just pass in the original function instead of making a lambda? That seems a little cleaner:
```suggestion
  memoized_fun.cache_clear = fun_caches.clear
```